### PR TITLE
fix Extract Indicators From File - Generic playbook when the pdf file is invalid

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16208

## Description
Extract Indicators From File - Generic breaks when a malformed image file with a pdf extension is passed to it.

## Related PRs
https://github.com/demisto/content/pull/3239

### CODE REVIEW

**Comment by yaakovi** [comment]
@adi88d test is failing

**Comment by JonathanMeler** [comment]
@yaakovi it's hard to follow a YMAL for changes.
Please let me know what exactly you did to fix the issue

**Comment by yaakovi** [comment]
only the condition task that determine if a file is a pdf or not:
File Info and File.Type should **contain** pdf (used to be equal "application/pdf")
there used to be an "or" between the conditions and not "and"

**Comment by JonathanMeler** [comment]
Sounds good
